chingsley/ireporter
# Ch update media upload

#### What does this PR do?
This pull request deals with implementation of the functionality to upload and save media on cloudinary.

#### Description of Task to be completed?
- In the server, create a cloudinary.js file.
- In the cloudinary.js file include the configuration for cloudinary as well as the class containing the functions that implement multiple and single uploads.
- Export the class from cloudinary.js.
- Import the exported class in the middleware files and use the class functions to save images and videos on cloudinary.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- ensure that you are a registered user. Then try creating a record (red-flag or intervention), this time, include files for upload. When you click the 'send' button, the files (images or video) will be saved on cloudinary, and these files can be seen when one clicks on the 'MY REPORTS' link.
